.gitignore

```diff
@@ -106,3 +106,8 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# IDEs
+/.vscode
+.idea
+*.iml
```

Comment:
Needs a new line.

Also, I _think_ we are supposed to use our local global gitignore for IDE stuff?

Reply:
Newline added.
We have stated adding those now, to better support new folks.